TRADEMARK-COMMITTEE.md

```diff
@@ -0,0 +1,145 @@
+   authority over Knative Trademark, conformance, and brand decisions.
+
+## Delegated authority
+
+The Trademark Committee may choose to create and delegate its authority to other
+committees as-needed. This may be used (for example) to create sub-committees
+for specific technical or marketing questions related to conformance or joint
+marketing.
+
+## Committee Meetings
+
+The committee meets as-needed. Meetings are held online.
+
+Given the private nature of some of these discussions (e.g. trade secrets, legal
+and compliance concerns, etc.) some meetings are held in private.
+
+Meeting notes are available to members of the
+[knative-dev mailing list](https://groups.google.com/forum/#!forum/knative-dev)
+(note: these are accessible to members of the knative-users list as well).
+Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as
+[issues in the community repo](https://github.com/knative/community/issues), and
+the trademark committee invites your feedback there. See
+[Getting in touch](#getting-in-touch) for other options.
+
+## Committee Members
+
+| Member                   | Organization | Profile |
+| ------------------------ | ------------ | ------- |
+| Unknown - to be selected | Google       | Unknown |
+| Unknown - to be selected | IBM/RedHat   | Unknown |
+| Unknown - to be selected | VMware       | Unknown |
+
+## Decision process
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding assignment of the mark, charter,
+conformance, and scope.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it. Vetoes by
+absent committee members must be presented within 5 business days of a published
+decision.
+
+Quorum is considered reached when at least half of the members are present.
+Quorum may be established over email or electronic chat systems as well as via
+video or in-person meetings.
+
+In case of extended absence or removal from the project for a code of conduct
+violation, the organization of the absent member may appoint a single
+replacement delegate from the same company during the absence.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
+
+1. Emailing the steering committee at
+   [knative-trademark@googlegroups.com](mailto:knative-trademark@googlegroups.com).
+   This is a private discussion list to which all members of the committee have
+   access.
+2. Open an issue on a knative repository and indicate that you would like
+   attention from the trademark committee.
+
+## Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year, as well as
+the trademark holder. These representatives shall be confirmed by the Knative
```

Comment:
s/These representatives shall be confirmed by the Knative Steering Committee; companies/Companies/

I don't think the KTC can afford to be in a position where it can't operate because KSC doesn't like its composition/doesn't confirm it.

Reply:
This was @mattmoor 's suggestion, here: https://github.com/knative/community/pull/277#discussion_r489559180

I don't have a dog in this race (and the catch-up clause I added here about delays in naming replacements makes it against KSC's interests to stall appointments, since that will simply stall KTC).

Removing for now, unless Matt wants to rally more community support.

I do echo his concern about lack of engagement from a KTC participant being a frustrating form of sabotage (if they have a veto but don't show up prepared, this could hamstring the trademark normalization process -- but we have the same problem today, so this reduces the scope of the problem).

Reply:
I reworked the catch-up clause. Given the current levels of trust, I wanted to make sure that we ring-fenced the appointments such that a temporary lapse in representation couldn't be abused for nefarious ends.

Reply:
I think the minimum bar here is agreeing that KTC is not immune from CoC violations and that KSC may remove individual representatives.  I would also expect that the remainder of KTC to not abuse this vacancy (in good faith) so long as it's clear that the company is (in good faith) trying to find a suitable replacement, and not just stall out the KTC.